.github/workflows/rebuild-docker-images.yaml

```diff
@@ -0,0 +1,82 @@
+on:
+    schedule:
+        -   cron: '0 3 * * *'
+
+name: Rebuild Docker images
+jobs:
+    rebuild-master-images:
+        name: Rebuild master images on Docker Hub
+        runs-on: ubuntu-20.04
+        steps:
+            -   name: GIT checkout branch - refs/heads/master
+                uses: actions/checkout@v2
+                with:
+                    ref: refs/heads/master
+            -   name: Login to Docker Hub
+                uses: docker/login-action@v1
+                with:
+                    username: ${{ secrets.DOCKER_USERNAME }}
+                    password: ${{ secrets.DOCKER_PASSWORD }}
+            -   name: Generate PHP-FPM tag
+                env:
+                    DOCKER_USERNAME: ${{ secrets.DOCKER_USERNAME }}
+                run: |
+                    DOCKER_PHP_FPM_IMAGE_TAG=gitlab-action-`find project-base/docker/php-fpm -type f -exec stat -c "%a %n" {} \; -exec cat {} \; | md5sum | awk '{ print $1 }'`
+                    echo "DOCKER_PHP_FPM_IMAGE_TAG=${DOCKER_PHP_FPM_IMAGE_TAG}" >> $GITHUB_ENV
+                    echo "DOCKER_PHP_FPM_MASTER_IMAGE_TAG=${DOCKER_PHP_FPM_IMAGE_TAG}" >> $GITHUB_ENV
+                    echo "DOCKER_USERNAME=${DOCKER_USERNAME}" >> $GITHUB_ENV
+            -   name: Build PHP-FPM image and push it to Docker Hub
+                run: |
+                    echo $PWD
+                    ls -la .github
+                    chmod +x .github/build-php-fpm-image.sh
+                    sh .github/build-php-fpm-image.sh
+            -   name: Generate Elasticseach tag
+                run: |
+                    DOCKER_ELASTICSEARCH_IMAGE_TAG=gitlab-action-`find project-base/docker/elasticsearch -type f -exec stat -c "%a %n" {} \; -exec cat {} \; | md5sum | awk '{ print $1 }'`
+                    echo "DOCKER_ELASTICSEARCH_IMAGE_TAG=${DOCKER_ELASTICSEARCH_IMAGE_TAG}" >> $GITHUB_ENV
+                    echo "DOCKER_ELASTICSEARCH_MASTER_IMAGE_TAG=${DOCKER_ELASTICSEARCH_IMAGE_TAG}" >> $GITHUB_ENV
+            -   name: Build Elasticsearch image and push it to Docker Hub
+                run: sh .github/build-elasticsearch-image.sh
+    rebuild-9-1-images:
```

Comment:
Ouch, I see it as too easy to forget about it. Unfortunately I have no idea how to do it more stable now.